source/_UIAHandler.py

```diff
@@ -361,7 +481,10 @@ def IUIAutomationFocusChangedEventHandler_HandleFocusChangedEvent(self,sender):
 			if _isDebug():
 				log.debug("HandleFocusChangedEvent: event received while not fully initialized")
 			return
+		previousFocusedUIAElement = self.lastFocusedUIAElement
 		self.lastFocusedUIAElement = sender
+		if self.localEventHandlerGroup:
+			self.MTAThreadQueue.put_nowait(lambda: self._onFocusChange(previousFocusedUIAElement, sender))

```

Comment:
Why does this have to be queued?  the focus event handler is already being executed in an MTA thread (granted it is probably not our specific one, but it is certainly running within the MTA apartment). Did you see problems not doing this?

Reply:
Honestly, I copied this behavior from #8742, but yeah, this implementation is different so that should work. I'll give it a spin.